maven-plugin/src/main/java/org/openmrs/maven/plugins/packager/config/CreateContentPackageMojo.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.maven.plugins.packager.config;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Properties;
+
+import static org.twdata.maven.mojoexecutor.MojoExecutor.configuration;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.element;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.executeMojo;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.goal;
+import static org.twdata.maven.mojoexecutor.MojoExecutor.plugin;
+
+/**
+ * The purpose of this Mojo is to support the migration away from config packages to content packages
+ * The intent is to bundle up the same configurations as content packages
```

Comment:
Config packages never lived outside of PIH, did they, or am I forgetting?  Might be worth referencing these as "PIH" config packages in the comments to help people 10 years from now remember what they were (or just realize they can ignore them)

Reply:
Oh, right, the packager plugin... maybe just reference that in the comments?

Reply:
Duh, sorry for all the spam... I was thinking this change was in the SDK... of course you don't need to reference the packager plugin since this is the package plugin... anyway, LGTM, ignore my comments...